state_mixin.py

```diff
@@ -14,6 +15,9 @@ class StateMixin(object):
     state_expr = ExpressionProperty(title='State Expression', default='{{$state}}')
     backup_interval = TimeDeltaProperty(title='Backup Interval',
                                         default={'seconds': 600})
+    load_from_persistence = BoolProperty(title="Load from persistence?", default=True,
```

Comment:
Since it gets added as part of this PR, it would be best to push another commit that removes it.

If it was an outstanding issue I would say leave it but since this is the pull that causes it we should take care of it